### ISSUE TITLE
Date Picker and Graph are out of Sync

### ISSUE THREAD

**Issue by LevinMedia** [woocommerce/wc-admin#374]
The x-axis ticks are out of sync by one day:

**Examples:**
<img width="1440" alt="428de562eed4cd0a8e0b67c76749d7cf _screen shot 2018-09-07 at 9 50 19 am" src="https://user-images.githubusercontent.com/4500952/45234155-bc0c8180-b289-11e8-81f7-bccf92a7a966.png">

<img width="1526" alt="screen shot 2018-09-07 at 10 29 23 am" src="https://user-images.githubusercontent.com/4500952/45234002-502a1900-b289-11e8-866b-ae7292f0c3f9.png">

In the second example above, I verified the reported revenue matches with the custom date selected in the date range picker. 

**Revenue on August 20th**
<img width="1495" alt="screen shot 2018-09-07 at 10 22 38 am" src="https://user-images.githubusercontent.com/4500952/45234072-7d76c700-b289-11e8-913a-dc9cbe539a60.png">

**Comment by psealock** [comment]
I can reproduce this locally and solve it by ensuring data points exist across all days, or at least the first and last interval.

I'm certain this problem will resolve itself upon zero-filling from the API, https://github.com/woocommerce/woocommerce/issues/21298. I'm going to close this issue and we can revisit if the problem pops up again after the API issue is resolved.

**Comment by LevinMedia** [comment]
@greenafrican re-opening this issue.

I did a little extra digging: Using smooth generator, I made sure I had a full month of data. I selected a week in the middle of the month (Aug 20 - 26). The chart was still shifted day off from the date range picker (Aug 19-25):

<img width="1350" alt="screen shot 2018-09-11 at 1 49 15 pm" src="https://user-images.githubusercontent.com/4500952/45387204-301c9180-b5ca-11e8-8468-8c46d5334742.png">

The API is sending dates that match the picker:

<img width="1384" alt="screen shot 2018-09-11 at 1 49 27 pm" src="https://user-images.githubusercontent.com/4500952/45387295-7ffb5880-b5ca-11e8-84ba-0ffc88ee1f8c.png">

However, I noticed it's using `date-start_gmt` which appears to be causing the issue. I changed my computer timezone to GMT (I'm currently in PST), reloaded the page, and the graph displayed the right days:

<img width="1343" alt="screen shot 2018-09-11 at 9 59 48 pm" src="https://user-images.githubusercontent.com/4500952/45387467-01eb8180-b5cb-11e8-950a-7f65d9641cfd.png">
